src/components/UploadButton.jsx

```diff
@@ -0,0 +1,36 @@
+import React from 'react'
+
+const styles = {
```

Comment:
We already debated on this in a previous PR, basically the reason is the component cannot possibly work witout these styles: these are "functional" styles, without them, the component is broken. That's why I think they belong to the component and not to an external stylesheet.